packages/transformers/js/native-browser.js

```diff
@@ -0,0 +1,37 @@
+import initFn, {transform} from './wasm/dist-web/parcel_js_swc_wasm.js';
+
+export const init = initFn();
+
+function transformWrapper(config) {
+  let result = transform(config);
+  return {
+    ...result,
+    // Hydrate Uint8Array into Buffer
+    code: Buffer.from(result.code.buffer),
+    // https://github.com/cloudflare/serde-wasm-bindgen/issues/10
+    dependencies: result.dependencies?.map(d => ({
+      ...d,
+      attributes:
+        d.attributes != null
+          ? Object.fromEntries([...d.attributes])
+          : undefined,
+    })),
+    hoist_result:
+      result.hoist_result != null
+        ? {
+            ...result.hoist_result,
+            imported_symbols: Object.fromEntries([
```

Comment:
Just noticed this while looking at the link from other PR and wanted to note here too that you can use `Object.fromEntries(result.hoist_result.imported_symbols)` directly without converting to an intermediate array.

Reply:
You're right. But even better, I can remove this altogether now: https://github.com/parcel-bundler/parcel/pull/6413